RecoPixelVertexing/PixelVertexFinding/src/gpuVertexFinderImpl.h

```diff
@@ -77,17 +119,37 @@ namespace gpuVertexFinder {
 #endif
 
 #ifdef __CUDACC__
-    if (useDensity_) {
-      clusterTracksByDensity<<<1, 1024 - 256, 0, stream>>>(soa, ws_d.get(), minT, eps, errmax, chi2max);
+    if (oneKernel_) {
+      // implemented only for density clustesrs   
+#ifdef ONE_KERNEL
+      vertexFinderOneKernel<<<1, 1024 - 256, 0, stream>>>(soa, ws_d.get(), minT, eps, errmax, chi2max);
```

Comment:
Is there a good motivation for the "one kernel" to be controlled both by `#ifdef` and by configuration parameter?

Reply:
ONE_KERNEL controls one vs three as before ONE was done using dynamic parallelism (some as three but in its own kernel) that at the moment does not even link...
I still hope at some point to have the possibility to test dynamic parallelism
(so ok, everything can be done at config level)
I can remove the three kernel option, is just historical development at this point...

Reply:
Ok, I was just curious. I'm fine with leaving it in, but maybe add a comment explaining `ONE_KERNEL`?

Reply:
I will cleanup and comment in next iteration.